src/panels/TcpDumpPanel.ts

```diff
@@ -76,7 +76,7 @@ export class TcpDumpDataProvider implements PanelDataProvider<"tcpDump"> {
         readonly kubectlVersion: KubectlVersion,
         readonly clusterName: string,
         readonly linuxNodesList: string[],
-    ) {}
+    ) { }
```

Comment:
Auto-formatting? We should probably standardize on code formatters. It should be using `prettier` for everything.